README.md

```diff
@@ -59,7 +59,37 @@ spec:
 
 This will create a new Sandbox named `my-sandbox` running the image you specify. You can then access the Sandbox using its stable hostname, `my-sandbox`.
 
-For more complex examples, including how to use the extensions, please see the [examples/](examples/) and [extensions/examples/](extensions/examples/) directories.
+For a more programatic approach you can use the [SDK](https://github.com/kubernetes-sigs/agent-sandbox/blob/main/clients/python/agentic-sandbox-client/README.md) like the following:
+```python
+from agentic_sandbox import Sandbox
+
+# The SDK abstracts all YAML into a simple context manager
+try:
+    with Sandbox(
+        template_name="python-runtime-template",
+        namespace="ai-agents"
+    ) as sandbox:
+
+        print("--- Sandbox is Ready! ---")
+
+        # 1. Run a command inside the secure sandbox
+        result = sandbox.run("echo 'Hello from inside the sandbox!'")
+        print(f"Stdout: {result.stdout.strip()}")
+
+        # 2. Write and read files
+        sandbox.write("test.txt", "This is a test file.")
```

Comment:
The `write()` method appears to take a string, while the `read()` method returns bytes that need decoding. This asymmetry in the API could be confusing. It would be clearer if the example showed encoding the string before writing, e.g., `sandbox.write("test.txt", "This is a test file.".encode('utf-8'))`, to make the process more explicit.